src/App.js

```diff
@@ -110,8 +112,9 @@ class App extends Component {
       renderFooterPagesList.includes(pathname) ? (
         <Footer />
       ) : null;
-    const renderCookiePolicy = showCookiePolicy ? <CookiePolicy /> : null;
-    const renderDialog = isModalOpen ? <DialogSection /> : null;
+    const renderCookiePolicy =
+      showCookiePolicy === true ? <CookiePolicy /> : null;
```

Comment:
Why is `=== true` required, if showCookiePolicy is already boolean?

Reply:
It is not working with
`const renderCookiePolicy = showCookiePolicy ? <CookiePolicy /> : null`